forge-ai/src/main/java/forge/ai/AiAttackController.java

```diff
@@ -152,6 +147,15 @@ public boolean apply(Card c) {
                     defenders.add(animatedCopy);
                 }
             }
+            // Transform (e.g. Incubator tokens)
+            for (SpellAbility sa : Iterables.filter(c.getSpellAbilities(), SpellAbilityPredicates.isApi(ApiType.SetState))) {
+                if ("Transform".equals(sa.getParam("Mode")) && ComputerUtilCost.canPayCost(sa, defender, false)
+                        && sa.getRestrictions().checkOtherRestrictions(c, sa, defender)) {
+                    Card transformedCopy = CardUtil.getLKICopy(c);
+                    transformedCopy.setState(CardStateName.Transformed, false);
+                    defenders.add(transformedCopy);
```

Comment:
might check if the transformed Copy is a creature?

Reply:
Yup, going to add that after work 👍

Reply:
Done :)